src/lib/setup.js

```diff
@@ -155,6 +157,8 @@ export default function (app) {
       _.extend({}, config, {
         CurrentUser: CurrentUser,
         ARTSY_URL: API_URL,
+        ARTSY_ID: CLIENT_ID,
+        ARTSY_SECRET: CLIENT_SECRET,
```

Comment:
Explicitly configure artsy-passport with `CLIENT_ID/CLIENT_SECRET` from env vars. artsy-xapp is [similarly configured](https://github.com/artsy/force/blob/5ecb9acc1777e4a077565946783077299cacd931/src/index.js#L124).